pyccel/ast/builtin_objects/list_functions.py

```diff
@@ -0,0 +1,39 @@
+# coding: utf-8
+#------------------------------------------------------------------------------------------#
+# This file is part of Pyccel which is released under MIT License. See the LICENSE file or #
+# go to https://github.com/pyccel/pyccel/blob/master/LICENSE for full license details.     #
+#------------------------------------------------------------------------------------------#
+"""
+The List container has a number of built-in methods that are 
+always available.
+
+In this module we implement List methods.
```

Comment:
I don't think the implementation of list methods is found in this file. Rather it contains objects which describe these methods within Pyccel's AST